test/CollapseSpec.js

```diff
@@ -174,7 +172,7 @@ describe('<Collapse>', () => {
     });
 
     it('Defaults to height', () => {
-      assert.equal(wrapper.instance().collapse.getDimension(), 'height');
+      assert.equal(collapseHelpers.getDimension(), 'height');
     });
 
     it('Uses getCollapsibleDimension if exists', () => {
```

Comment:
**nitpick:** this test case could probably use a better name... probably just "uses dimension if exists", given that there's no longer any `getCollapsibleDimension`